ksql-cli/src/main/java/io/confluent/ksql/cli/console/JLineReader.java

```diff
@@ -49,6 +52,18 @@
    */
   private static class KsqlExpander extends DefaultExpander {
 
+    private static final Map<String, String> shortcuts = Maps.newHashMap();
+    
+    {
+      shortcuts.put("cs", "CREATE STREAM s ( field1 type1 ) WITH (KAFKA_TOPIC='topic',"
```

Comment:
maybe use stream/table instead of s/t/x, so its more clear what those are supposed to refer to.

Reply:
Thanks @rodesai for the feedback! I'd actually gone back and forth on that idea while testing - there's something of a trade-off here between making it as super-obvious as possible what everything is and reducing the number of characters which need to be deleted/over-typed to get to your desired end-state. If you still feel strongly about i'm happy to change though...

Reply:
No worries. Just wanted to make sure it was considered.